app/src/main/assets/licenses.xml

```diff
@@ -57,27 +75,33 @@
     <library
         name="jsoup"
         author="Jonathan Hedley"
-        website="http://jsoup.org/"
+        website="https://jsoup.org/"
         license="MIT"
         licenseText="LICENSE_JSOUP.txt" />
     <library
-        name="Material Design Icons"
+        name="Lightweight-Stream-API"
+        author="Victor Melnik"
+        website="https://github.com/aNNiMON/Lightweight-Stream-API"
+        license="Apache 2.0"
+        licenseText="LICENSE_APACHE-2.0.txt" />
+    <library
+        name="Material Components for Android"
         author="Google"
-        website="https://github.com/google/material-design-icons"
-        license="Attribution-ShareAlike 4.0 International"
-        licenseText="LICENSE_MATERIAL_DESIGN_ICONS.txt" />
+        website="https://github.com/material-components/material-components-android"
+        license="Apache 2.0"
+        licenseText="LICENSE_APACHE-2.0.txt" />
     <library
         name="Material Design Icons"
-        author="Templarian"
-        website="https://github.com/Templarian/MaterialDesign"
-        license="SIL Open Font, Version 1.1"
-        licenseText="LICENSE_SIL.txt" />
-    <library
-        name="Material Components"
         author="Google"
-        website="https://github.com/material-components/material-components-android"
+        website="https://github.com/google/material-design-icons"
         license="Apache 2.0"
         licenseText="LICENSE_APACHE-2.0.txt" />
+    <library
+        name="Material Design"
```

Comment:
I would include the word "icons" here. Otherwise, it is a bit misleading. This is the license of the icons, not the license of material design as a whole.

Reply:
I tried to change that one up because "Material Design Icons" was already in the list, so I felt it would be weird it if looked like there was a duplicate. Is there possibly an alternative name that would work? Maybe something like this?

`Material Design Icons (by Google)`

`Material Design Icons (custom)`

Reply:
The list shows the authors directly below, so I don't think that needs to be stated again

Reply:
Mm, alright.

Reply:
I agree it's not optimal but I think it is better than the two alternatives (misleading or duplicate information)